GermanFilter/sections/specific.txt

```diff
@@ -9,6 +9,8 @@
 !
 ! https://github.com/AdguardTeam/AdguardFilters/issues/61907
 ||chip.de/*.php*&$~third-party,script
+! https://github.com/AdguardTeam/AdguardFilters/issues/76267
+karrieresprung.de##div[id^="ks_ad_"]
```

Comment:
please move under this line https://github.com/AdguardTeam/AdguardFilters/blob/eb36a9ce2e1972dab48157d166086f9197000251/GermanFilter/sections/specific.txt#L78

Reply:
Moved.

Reply:
Thanks.